plugin.audio.vrt.radio/addon.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.vrt.radio" name="VRT Radio" version="0.1.0" provider-name="dagwieers">
+    <requires>
+        <!-- import addon="resource.images.studios.coloured" version="0.0.15"/ -->
+        <import addon="resource.images.studios.white" version="0.0.21"/>
+        <import addon="script.module.routing" version="0.2.0"/>
+        <import addon="xbmc.python" version="2.25.0"/>
```

Comment:
WARN: For leia it is advised to set xbmc.python version to 2.26.0
```suggestion
        <import addon="xbmc.python" version="2.26.0"/>
```

Reply:
But the plugin also works on krypton. So why artificially limit to Leia here, if it will only ship to Leia? It means I have to remember changing this for this repository ever release.